composer.json

```
@@ -12,12 +12,12 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "dgame/php-ensurance": "^1.0 || ^2.0",
-        "ralouphie/mimey": "^2.0"
+        "php": ">=7.4",
```

Nicht das? :stuck_out_tongue: 
```suggestion
        "php": "^8.1",
```

---

Es gibt kein festes Abhängigkeit auf 8.1 :shrug: